models/tamu/PromptDs_TAMU_RAA_5TeV_010.txt

```diff
@@ -1,34 +1,63 @@
 # 0-10%, Ds-meson R_AA
```

Comment:
maybe for the TAMU model would be good to still keep the old version because for non-prompt Ds mesons we only have the old version (but maybe not needed to keep compatibility between prompt and nonprompt when computing pT weights? Not sure)

Reply:
I would use the new version of TAMU for the prompt pT weights also in the non-prompt analysis. We know that this is better so no reason to use the old one.